index.html

```diff
@@ -1,41 +1,217 @@
+                  <li>Javascript</li>
+                  <li>Ruby</li>
+                  <li>Html</li>
+                  <li>Css</li>
+                </ul>
+              </div>
+            </li>
+            <li>
+              <h4>Frameworks</h4>
+              <div class="inner-list-container">
+                <ul>
+                  <li>Bootstrap</li>
+                  <li>Ruby on Rails</li>
+                  <li>RSPec</li>
+                  <li>CapyBara</li>
+                  <li>Selenium</li>
+                </ul>
+              </div>
+            </li>
+            <li>
+              <h4>Skills</h4>
+              <div class="inner-list-container">
+                <ul>
+                  <li>Codekit</li>
+                  <li>Github</li>
+                  <li>Codepen</li>
+                  <li>Gitlab</li>
+                  <li>Terminal</li>
+                </ul>
+              </div>
+            </li>
+          </ul>
+          <img src="images/image4.png" alt="border" class="image-border" />
+        </section>
+      </div>
+      <div class="section3-details">
+        <section>
+          <div class="social-container">
+            <a>
+              <i class="fa fa-twitter" aria-hidden="true"></i>
+            </a>
+            <a>
+              <i class="fa fa-github" aria-hidden="true"></i>
+            </a>
+            <a>
+              <i class="fa fa-linkedin" aria-hidden="true"></i>
+            </a>
+            <a>
+              <i class="fa fa-medium" aria-hidden="true"></i>
+            </a>
+          </div>
+          <div class="work-icon">
+            <i class="fa fa-arrow-down" aria-hidden="true"></i>
+            <h3>Work</h3>
+          </div>
+          <h1>MY RECENT WORK</h1>
+          <img src="images/image4.png" alt="border" class="image-border" />
+        </section>
+      </div>
+      <div class="section4-work-details">
+        <section>
+          <div class="personal-projects">
+            <img src="images/image2.png" alt="project-image" />
+            <h1>Multi-Post Stories</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li class="list-sumbua">
+                CSS <span> <img src="images/image3.png" alt="css" /></span>
+              </li>
+              <li class="list-sumbua">
+                Html <span> <img src="images/image3.png" alt="html" /></span>
+              </li>
+              <li class="list-sumbua">
+                Bootstrap
+                <span> <img src="images/image3.png" alt="bootstrap" /></span>
+              </li>
+              <li>Ruby</li>
+            </ul>
+            <button class="seeproject-btn" type="button">See Project</button>
+          </div>
```

Comment:
- [ ] Kindly implement the spacing equally from both sides left and right to stick with the Figma design. Implement in all the section
- your design
![his design](https://user-images.githubusercontent.com/81727988/164730598-63d3658b-765b-467d-bc5e-af17e4eca0b0.png)

- Figma design
![figma](https://user-images.githubusercontent.com/81727988/164730678-acf53360-21fd-4425-ae5d-9df12031a4e8.png)